android/assets/jsons/translations/German.properties

```diff
@@ -638,11 +638,11 @@ Clearing a [forest] has created [amount] Production for [cityName] = Die Rodung
 The [questName] quest for [civName] has ended. It was won by [civNames]. = Die Aufgabe [questName] für [civName] ist beendet. Sie wurde von [civNames] erledigt.
 The resistance in [cityName] has ended! = Der Widerstand in [cityName] ist beendet!
  # Requires translation!
-[cityName] demands [resource]! = 
+[cityName] demands [resource]! = [cityName] fordert [resource]!
  # Requires translation!
-Because they have [resource], the citizens of [cityName] are celebrating We Love The King Day! = 
+Because they have [resource], the citizens of [cityName] are celebrating We Love The King Day! = Weil sie [resource] haben, feiern die Einwohner von [cityName] den Wir Lieben den König Tag
```

Comment:
Satzzeichen fehlt
"Wir Lieben den König Tag" sollte eher, wie in der englischen Vorlage "Wir Lieben Den König Tag" geschrieben werden.